### PR TITLE
tests: add doc to describe how to run conformance tests in a running cluster

### PR DESCRIPTION
**Pull request by cpanato**
@sym3tri can you please review? thanks 😄

### CODE REVIEW

**Comment by sudhaponnaganti** [comment]
@cpanato - Would it be possible to add a section how the nightly runs are working. 
Is the image building happening automatically now? If so it would be good to add that as well i,e how   image is being built as well even if that is automated.

**Comment by cpanato** [comment]
@sudhaponnaganti 
 - will add a nightly section.
 - the conformance docker image is build manually today, we can create a Jenkins job to build and push the image to quay.

File: tests/README.md [hunk 1]
```diff
@@ -152,3 +160,56 @@ TF_VAR_tectonic_gcp_ssh_key
...
+* Kube Config file
+
+Then, you need to execute the following command in your shell:
+
+```Bash
+$ docker run -v <KUBECONFIG_PATH>:/kubeconfig quay.io/coreos/kube-conformance:<TAG>
+```
+
+where:
+`<TAG>` is the version of the Kubernetes we are using, i.e, `v1.8.2_coreos.0`
```
Comment (sym3tri) [inline tests/README.md]: With respect to the Tectonic, the version mentioned here corresponds to the same tag used in the cluster on the Hpyerkube image. It's worth mentioning Hyperkube (add link to its quay.io hyperkube image).

---

Also, can you add more details about the version requirements?

i.e. Does the test image tag _always_ have to match _exactly_ the hyperkube image tag version?

**Comment by sym3tri** [comment]
1 comment, otherwise LGTM

File: tests/README.md (hunk 1 above)
Comment (cpanato) [reply tests/README.md]: did my best

**Comment by cpanato** [comment]
ok to test

**Pull request merged**